tests/format/js/sloppy-mode/function.js

```diff
@@ -0,0 +1 @@
+foo: function bar() {}
```

Comment:
Let's name this file as `labeled-function-declaration.js` ?

Reply:
`labeled-function-declaration.js` sounds good